decidim-core/lib/decidim/traceable.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # A concern that adds traceabilty capability to the given model. Including this
+  # allows you the keep track of changes in the model attributes and changes authorship.
+  #
+  # Example:
+  #
+  #     class MyModel < ApplicationRecord
+  #       include Decidim::Traceable
+  #     end
+  module Traceable
+    extend ActiveSupport::Concern
+
+    included do
+      has_paper_trail
+
+      delegate :count, to: :versions, prefix: true
+
+      def last_whodunnit
+        versions.last.whodunnit
```

Comment:
This doesn't appear as covered by the test suite. Maybe we're lacking a test?

Reply:
Yes, it's not covered by the test suite because I considered it was pretty straightforward, I'll add it.